README.md

```diff
@@ -244,6 +246,47 @@ function App() {
 }
 ```
 
+## Manual Cancellation
```

Comment:
if you add this section please link it in the TOC in the "guides" section. I would also appreciate if you could include a codesandbox rather than a long example in the readme, or both

Reply:
Will do. 

I'd like to include a sandbox, but i'm having issues with importing the latest changes ( in master branch) of axios-hooks into it. Do I have to wait for a release? Or do you know if there's a way to make it work such as : 
```js
// package.json
"axios-hooks": "https://github.com/simoneb/axios-hooks",
```
```js
// index.js
import { makeUseAxios } from 'axios-hooks';
```

Reply:
ah you're right, you can't actually create the codesanbox example until a new version is released. let's get this out then we can create a sandbox example and include it in the readme

Reply:
Linked it in the Table of contents. After the next release i can add the other links to the sandbox.